### DISCUSSION TITLE
Show symlinks with different color than normal files/directories

### DISCUSSION THREAD

**Discussion by LinArcX** [sxyazi/yazi#594] [4. Ideas]
If you apply this tip: https://yazi-rs.github.io/docs/tips#show-symlink-in-status-bar

You can't distinguish symlink directories from normal directories and you should always look at the bottom to see the difference. one of the cool features of `lf` was that by default it shows symlinks directories with different colors. maybe we can have this feature via an option? (also we can have another option to show symlinks as italics or normal)

**Comment by og900aero** [comment]
Use theme.toml file, and insert the following line after [filetype]  rules section:

{ name = "*/", is = "link",  fg = "yellow" },

Or use color code instead of yellow.

**Reply by LinArcX** [reply]
Thanks a lot.

**Comment by sxyazi** [comment]
Yeah the doc is here https://yazi-rs.github.io/docs/configuration/theme/#filetype
